### PR TITLE
stbt.ocr: Make the default value of `lang` configurable

### PR DESCRIPTION
**Pull request by drothlis**
To avoid specifying it in every single call to `stbt.ocr`.

### CODE REVIEW

File: stbt/__init__.py [hunk 1]
```diff
@@ -501,7 +501,9 @@ def is_screen_black(frame=None, mask=None, threshold=None):
       binary threshold is applied to the frame. The ``threshold`` value is in
       the range 0 (black) to 255 (white). The global default can be changed by
       setting ``threshold`` in the ``[is_screen_black]`` section of
-      :ref:`.stbt.conf`.
+      :ref:`.stbt.conf`. You may need to install the tesseract language pack;
+      see installation instructions `here
+      <https://stb-tester.com/manual/troubleshooting#install-ocr-language-pack>`_.
```
Comment (wmanley) [inline stbt/__init__.py]: This is useful documentation.  We could even have a lint for it, but it probably wouldn't be worth the effort.

File: stbt/__init__.py [hunk 2]
```diff
@@ -108,7 +108,7 @@ def press(key, interpress_delay_secs=None):
 
         This defaults to 0.3. You can override the global default value by
         setting ``interpress_delay_secs`` in the ``[press]`` section of
-        stbt.conf.
+        :ref:`.stbt.conf`.
```
Comment (wmanley) [inline stbt/__init__.py]: A good usability improvement.

File: stbt/__init__.py (hunk 1 above)
Comment (wmanley) [reply stbt/__init__.py]: This is in the wrong docstring.

Comment (drothlis) [reply stbt/__init__.py]: Oops moved that to `stbt.ocr` docstring.

**Pull request merged**